daras_ai_v2/billing.py

```
@@ -474,7 +472,7 @@ def render_all_plans(
 
     with gui.div(className="my-2 d-flex justify-content-center"):
         gui.caption(
-            f"**[See all features & benefits]({settings.PRICING_DETAILS_URL})** • Prices don’t include taxes",
+            f"**[See all features & benefits]({settings.PRICING_DETAILS_URL})** • Prices don’t include taxes • Questions? Mail [humans@gooey.ai](mailto:humans@gooey.ai)",
```

Ruff (0.15.10)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)